website/content/docs/platform/k8s/helm/enterprise.mdx

```diff
@@ -46,6 +48,38 @@ Once the cluster is [initialized and unsealed](/docs/platform/k8s/helm/run), you
 kubectl exec -ti vault-0 -- vault license get
 ```
 
+### License Update
+
+To update the autoloaded license in Vault, you may do the following:
+
+- Delete and recreate the secret
+
+  Use the name specified in chart option [`server.enterpriseLicense.secretName`](/docs/platform/k8s/helm/configuration#secretname-1). Updating the existing secret will work as well.
+
+```shell
+kubectl delete secret vault-ent-license
+secret=$(cat new-license.hclic)
+kubectl create secret generic vault-ent-license --from-literal="license=${secret}"
+```
+
+- Wait until [`vault license inspect`](/docs/commands/license/inspect) shows the updated license
```

Comment:
The fact this would show the updated license before `vault license get` confused me a bit until I read the `vault license inspect` docs. Is it worth adding that this will show the updated license once the container's environment variable has been updated?

Reply:
That confused me at first too. Tried to clarify in d3e7aa8183defd0e013a20774ea3f32a3965550d.